### ISSUE TITLE
search giving wrong page

### ISSUE THREAD

**Issue by PureDreamer** [barrust/mediawiki#87]
```python
from mediawiki import DisambiguationError, MediaWiki
wikipedia = MediaWiki()
search = "Mother"
meaning = wikipedia.page(search)
print(meaning)
def_meaning = meaning.summarize(chars=300)
print(meaning)
print(def_meaning)
```
this will result in:
<MediaWikiPage 'Father'>
<MediaWikiPage 'Father'>
A father is the male parent of a child. Besides the paternal bonds of a father to his children, the father may have a parental, legal, and social relationship with the child that carries with it certain rights and obligations. An adoptive father is a male who has become the child's parent through the...

why does it gives the father page instead of the mother?

**Comment by barrust** [comment]
This issue seems to have been resolved in a previous [pull request](https://github.com/barrust/mediawiki/pull/85). This hasn't been released yet to pypi so you are likely using an out of date version. I need to publish a new version of pymediawiki which should resolve this issue. When running your code on the latest version I get the page for "mother" and not "father".

**Comment by barrust** [comment]
I have published a new release: `0.6.6` that should resolve this issue. If it does not, please reopen! 

``` bash
pip install pymediawiki --upgrade
```
